### DISCUSSION TITLE
Is it possible to migrate from single LND to LITD plus integrated LND?

### DISCUSSION THREAD

**Discussion by irusensei** [lightninglabs/lightning-terminal#724] [Q&A] [answered]
Is this possible or safe? The folder structure seems slightly different. 

Thanks.

**Comment by guggero** [comment, marked as answer]
Yes, it's possible. See https://github.com/lightninglabs/lightning-terminal/blob/master/doc/config-lnd-integrated.md#upgrade-existing-nodes.
The guide hasn't been updated in a while but should still work. Let us know if you run into any issues.

**Reply by irusensei** [reply]
That's a good start. Thank you!
